homeassistant/components/hassio/ingress.py

```diff
@@ -223,39 +232,28 @@ def _init_header(request: web.Request, token: str) -> CIMultiDict | dict[str, st
     # Set X-Forwarded-Proto
     forward_proto = request.headers.get(hdrs.X_FORWARDED_PROTO)
     if not forward_proto:
-        forward_proto = request.url.scheme
+        forward_proto = request.scheme
```

Comment:
Calling `.url` would build a new URL object using `.scheme` and throw it away and return the same scheme